### DISCUSSION TITLE
More levels in top navigation

### DISCUSSION THREAD

**Discussion by drfurtado** [quarto-dev/quarto-cli#695] [Q&A] [answered]
My site has the following directory structure:

Home
Teaching
----Course1 > index.qmd 
----Course2 > index.qmd
Research
About
...

How do set up the top navigation using the dropdown menu? I do not see in the docs a reference for two or more levels.

Any help is greatly appreciated.

**Comment by jjallaire** [comment, marked as answer]
Creating submenus is documented here: https://quarto.org/docs/websites/website-navigation.html#top-navigation

**Reply by drfurtado** [reply]
Thanks for the quick reply. I studied the content under the link you sent, but it refers to creating menus that link to files in the root directory, not inside sub-menus.  

I want to create a menu on the top bar with the following structure

Level1 (Teaching)
... Level2 (Course1) > linkto index.qmd inside a directory called "Course1"
... Level2 (Course2) > linkto index.qmd inside a directory called "Course2"
Level 1 (Research)
...

Any help would be greatly appreciated.

**Reply by dragonstyle** [reply]
You can see a pretty complex, multi-level top and side navigation example on the Quarto website itself. For example this page:

https://quarto.org/docs/websites/website-tools.html

is in the websites folder, but appears in the third level of the sidebar navigation. You can see how this defined the `_quarto.yml` for the website here:

https://github.com/quarto-dev/quarto-web/blob/main/_quarto.yml

HTH!
